src/winzone/WindowsTimeZoneIDs.jl

```diff
@@ -2,11 +2,19 @@ module WindowsTimeZoneIDs
 
 using ...TimeZones: DEPS_DIR
 using EzXML
+if VERSION >= v"1.4"
+    using Pkg.Artifacts
+else
+    # to avoid fail when Julia tried to precompile even non-compatible code
+    macro artifact_str(name)
+        :(throw("this should never bee called"))
+    end
+end
 
 # A mapping of Windows timezone names to Olson timezone names.
 # Details on the contents of this file can be found at:
 # http://cldr.unicode.org/development/development-process/design-proposals/extended-windows-olson-zid-mapping
-const WINDOWS_ZONE_URL = "http://unicode.org/repos/cldr/trunk/common/supplemental/windowsZones.xml"
+const WINDOWS_ZONE_URL = "https://raw.githubusercontent.com/unicode-org/cldr/latest/common/supplemental/windowsZones.xml"
```

Comment:
My bad, this should be master and not latest
```suggestion
const WINDOWS_ZONE_URL = "https://raw.githubusercontent.com/unicode-org/cldr/master/common/supplemental/windowsZones.xml"
```

Reply:
This change should also resolve the conflict